.travis.yml

```diff
@@ -42,6 +42,8 @@ before_install:
     esac
 
     pip install pylint
+    pip install grpcio==1.4.0
+    pip install portpicker==1.1.1
```

Comment:
This dependency is so trivial that it shouldn't be difficult to add to our Bazel build.